src/requirements.txt

```diff
@@ -1,3 +1,5 @@
 scrapy==1.0.5
 SQLAlchemy
 beautifulsoup4>=4.0
+ipdb
```

Comment:
For debugging. It's pdb on steroids - opens the debugger in an ipython console with autocomplete and all the goodies.